homeassistant/components/esphome/voice_assistant.py

```diff
@@ -307,24 +217,11 @@ async def run_pipeline(
             "raw" if self.device_info.voice_assistant_version >= 2 else "mp3"
         )
 
-        if use_vad:
```

Comment:
What about the VAD to detect the end of a command? Do we need to indicate to Assist that this is necessary ?

Reply:
The assist_pipeline always does this now AFAICS

https://github.com/home-assistant/core/blob/c17f08a3f543e63da66d5a10156776ee150c5f11/homeassistant/components/assist_pipeline/pipeline.py#L608-L624